src/pages/ShareCodePage.tsx

```diff
@@ -22,17 +24,38 @@ import * as Url from '@libs/Url';
 import * as UserUtils from '@libs/UserUtils';
 import CONST from '@src/CONST';
 import ROUTES from '@src/ROUTES';
-import type {Report} from '@src/types/onyx';
+import type {Policy, Report} from '@src/types/onyx';
 
 type ShareCodePageOnyxProps = {
     /** The report currently being looked at */
     report?: OnyxEntry<Report>;
+
+    /** The policy for the report currently being looked at */
+    policy?: OnyxEntry<Policy>;
 };
 
 type ShareCodePageProps = ShareCodePageOnyxProps;
 
-function ShareCodePage({report}: ShareCodePageProps) {
+/**
+ * When sharing a policy (workspace) only return user avatar that is user defined. Default ws avatars have separate logic.
+ * In any other case default to expensify logo
+ */
+
+function getLogoForWorkspace(report: OnyxEntry<Report>, policy?: OnyxEntry<Policy>): ImageSourcePropType | undefined {
+    if (!policy || !policy.id || report?.type !== 'chat') {
+        return expensifyLogo;
+    }
+
+    if (!policy.avatarURL) {
+        return undefined;
+    }
```

Comment:
no I don't think so, because if there is no avatarURL there can still be the default workspace image right?
so then this conditional catches the `undefined` value: https://github.com/Expensify/App/pull/39072/files#diff-1aedd048e3c292396a3cdb4db0965f01fe5324f2f79a3df05e1f9798aa5c1bf9R90

And it will then run logic to generate the default (SVG) avatar. At least that is my understanding of what should happen.

Reply:
Ah yea, makes sense. Thanks for clarifying!